learn10-cfn.yml

```
@@ -0,0 +1,521 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Lecture10 Create Base from CFn
+
+Parameters: 
+  cfnBase:
+    Description: this is VPC base name.
+    Type: String
+    Default: fromCFn
+  VPCcider:
+    Type: String
+    Default: 10.0.0.0/16
+  PBSubnetcider01:

```

cider -> CIDRが一般的ですね！
```suggestion
  PBSubnetCIDR01:
```

---

大変失礼致しました！
こちら修正致しました！